PactNet/IPactBuilder.cs

```diff
@@ -8,10 +8,10 @@ public interface IPactBuilder
     {
         IPactBuilder ServiceConsumer(string consumerName);
         IPactBuilder HasPactWith(string providerName);
-        IMockProviderService MockService(int port, bool enableSsl = false, IPAddress host = IPAddress.Loopback, string sslCert = null, string sslKey = null, bool useRemoteMockService = false);
+        IMockProviderService MockService(int port, bool enableSsl = false, bool enableIpv6 = true, IPAddress host = IPAddress.Loopback, string sslCert = null, string sslKey = null, bool useRemoteMockService = false);

```

Comment:
This is a breaking API change and I don't think we're looking to do another major release before moving to the Rust backend instead of Ruby

Reply:
I will arrange to move the enableIpv6 flag to the end, after the useRemoteMockService flag. That should avoid a breaking change.